### ISSUE TITLE
Icons showing even when spotify is closed

### ISSUE THREAD

**Issue by tiklup11** [koolskateguy89/gnome-shell-extension-spotify-controller#15]
Was working fine 2-3 weeks before. But now icons are there. It is turned off in settings.

**Comment by koolskateguy89** [comment]
- What GNOME version are you on?
- What version of the extension are you on?
- What do you mean by `now icons are there`?

**Comment by Erik0806** [comment]
I have the same issue,
I'm on Gnome version 42
and the newest version of the extension that is on extensions.gnome.org (I think it is 11)
As you start the laptop the control buttons are showing up (spotify is not starting automatically).
As soon as you start spotify and then close it again it disappears and works correctly.

**Comment by sergiotca** [comment]
Gnome Shell 42.4  -   X11  -  Extension v11

I can confirm this behavior persists on latest shell and extension (buttons show up even when Spotify is closed):
![image](https://user-images.githubusercontent.com/41351896/184716333-2a18e926-8a53-42fa-a385-f66921efbe46.png)
They do go away once you open and close spotify, but I've noticed they tend to come back whenever I use any other media player.

I've also noticed the "play" button changes to the "pause" button if when something (not spotify) is playing, but this doesn't mean you can use it to control anything else than spotify, as it should I guess. 
In this example I was playing a podcast from my phone through bluetooth (not on the spotify app either btw):![Captura desde 2022-08-15 15-53-18](https://user-images.githubusercontent.com/41351896/184718338-478e9923-b675-45f7-b6ab-8ad5a2d1ddac.png)

**Comment by koolskateguy89** [comment]
I've finally been able to reproduce this and know why this is happening:

- `spotify.PlaybackStatus` is not null when the `PlaybackStatus` of another media player (e.g. `vlc.PlaybackStatus` changes, even though there is no process taking ownership of the Spotify's well-known name (i.e. Spotify isn't open)
- because the extension checks if Spotify is open by checking `spotify.PlaybackStatus` is not null, it thinks Spotify is open

I think I've fixed this by instead listening for when Spotify's name appears/disappears (i.e. when Spotify opens/closes), I'll push soon
